Content.Server/Anomaly/Components/ReagentProducerAnomalyComponent.cs

```diff
@@ -121,6 +107,12 @@ public sealed partial class ReagentProducerAnomalyComponent : Component
     [DataField, ViewVariables(VVAccess.ReadOnly)]
     public ProtoId<ReagentPrototype> ProducingReagent = "Water";
 
+    /// <summary>
```

Comment:
store this on the system not component
or make it a component field...

also standar**d** not standart

Reply:
Since the system is the same for all anomalies, how to store the reagent produced by a particular anomaly in it? Isn't this an ECS violation?

Reply:
> or make it a component field...

Isn't it a field? Should I make it Read Write?

Reply:
make it ReadWrite 🤷

Reply:
> Since the system is the same for all anomalies, how to store the reagent produced by a particular anomaly in it? Isn't this an ECS violation?

the fallback reagent not ProducingReagent :trollface:

Reply:
@deltanedas 
![image](https://github.com/space-wizards/space-station-14/assets/96445749/950e557e-5dcc-4b3f-9291-3e6f036df72a)
like this?

Reply:
it shouldnt be a datafield if its const

Reply:
he's not now.

Reply:
What do I need to rewrite in this constant to make it a component field?
What??
😱

Reply:
oh
I got it. Constants can be written into the system. That's right.

Reply:
done.